### PR TITLE
small addition to document expected behavior of parallel autobuilds

### PR DESCRIPTION
**Pull request by ingshtrom**
### Proposed changes

There was no documentation on the expected behavior of parallel builds, which is a line item on our pricing page.

### CODE REVIEW

File: docker-hub/builds/index.md [hunk 1]
```diff
@@ -31,6 +31,11 @@ pushing to the registry. You can use these tests to create a continuous
 integration workflow where a build that fails its tests does not push the built
 image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
 
+Depending on your plan, you may get parallel builds, which means that `N`
+autobuilds can be run at the same time. `N` is configured by the plan that you
+subscribe to. Once `N+1` builds are running, any builds additional builds will
```
Comment (usha-mandya) [inline docker-hub/builds/index.md:36]:
```suggestion
subscribe to. Once `N+1` builds are running, any additional builds will
```

**Comment by usha-mandya** [comment]
Thank you @ingshtrom. Noted a minor update. Looks good otherwise.

File: docker-hub/builds/index.md [hunk 2]
```diff
@@ -31,6 +31,11 @@ pushing to the registry. You can use these tests to create a continuous
 integration workflow where a build that fails its tests does not push the built
 image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
 
+Depending on your plan, you may get parallel builds, which means that `N`
+autobuilds can be run at the same time. `N` is configured by the plan that you
+subscribe to. Once `N+1` builds are running, any additional builds will
+be queued up to be run later.
```
Comment (thaJeztah) [inline docker-hub/builds/index.md:37]: nit: perhaps avoid future tense

```suggestion
subscribe to. Once `N+1` builds are running, any additional builds are
queued to be run later.
```

**Review by usha-mandya — APPROVED** [review]

File: docker-hub/builds/index.md [hunk 3]
```diff
@@ -31,6 +31,11 @@ pushing to the registry. You can use these tests to create a continuous
 integration workflow where a build that fails its tests does not push the built
 image. Automated tests do not push images to the registry on their own. [Learn more about automated image testing here.](automated-testing.md)
 
+Depending on your plan, you may get parallel builds, which means that `N`
+autobuilds can be run at the same time. `N` is configured by the plan that you
+subscribe to. Once `N+1` builds are running, any additional builds are
+queued to be run later.
```
Comment (arkodg) [inline docker-hub/builds/index.md:37]:
```suggestion
queued to be run later.  The total number of pending builds in the queue is limited to 30 and further requests will be
ignored
```

Comment (usha-mandya) [reply docker-hub/builds/index.md:37]: Thanks Arko. I'll do a follow-up PR for this update.

Comment (arkodg) [reply docker-hub/builds/index.md:37]: thanks maybe we can use `discarded` instead of `ignored`

Comment (usha-mandya) [reply docker-hub/builds/index.md:37]: @arkodg https://github.com/docker/docker.github.io/pull/11689 - PTAL

**Pull request merged**